src/app/payment/services/checkout.service.ts

```diff
@@ -158,6 +155,11 @@ const checkoutSetupIntent = async (customerId: string) => {
   }
 };
 
+const verifyCryptoPayment = async (token: PaymentIntentCrypto['token']): Promise<boolean> => {
```

Comment:
this is not being used, the flow to verify the payments comes in another PR?

Reply:
Yep. I just added all the necessary logic in this PR. We will use it later.